### DISCUSSION TITLE
The screen color gamut type automatically changes

### DISCUSSION THREAD

**Discussion by xsx200011** [MonitorControl/MonitorControl#1006] [General]
### Before opening the issue, have you...?

- [ ] Searched for existing issues

### Question

In the initial state, select Mac mode (P3 gamut) by pressing the button on the display, then open the software and adjust the brightness. After success, open the display Settings, and find that the original SRGB gamut is automatically changed. After modification of the Settings, this problem still exists

**Comment by waydabber** [comment]
Does this happen on the (unknown) display? If so, then probably upon receiving a DDC command the display firmware might do something like this. Since this is a firmware issue, there is no much to do about it. You can disable DDC control and use purely software brightness control which cannot cause an issue like this with the display.

Hope this helps!
